openhtf/io/frontend/__main__.py

```diff
@@ -52,33 +50,14 @@
 from openhtf.util import multicast
 
 
-FLAGS = gflags.FLAGS
-
-gflags.DEFINE_integer('port', 12000,
-                      'The port on which to serve the frontend.')
-gflags.DEFINE_integer('poll_interval_ms', 3000,
-                      'Desired time between frontend polls in milliseconds.')
-gflags.DEFINE_integer('discovery_interval_s', 5,
-                      'Seconds to wait between station discovery attempts.')
-gflags.DEFINE_string('discovery_address',
-                     multicast.DEFAULT_ADDRESS,
-                     'Multicast address to ping for station discovery.')
-gflags.DEFINE_integer('discovery_port',
-                      multicast.DEFAULT_PORT,
-                      'Multicast port to use for station discovery.')
-gflags.DEFINE_integer('discovery_ttl',
-                      multicast.DEFAULT_TTL,
-                      'TTL for station discovery pings.')
-
-
 _LOG = logging.getLogger(__name__)
 
 UNKNOWN_STATION_ID = 'UNKNOWN_STATION'
+BUILD_PATH = os.path.join(os.path.dirname(__file__), 'src', 'dist')
+PREBUILT_PATH = os.path.join(os.path.dirname(__file__), 'prebuilt')
```

Comment:
I added a glob to `package_data` in `setup.py` that should grab the whole prebuilt, so this logic _should_ work both when developing/running from source, and when installed via a wheel.